website/docs/main/Avatar.md

```diff
@@ -0,0 +1,267 @@
+---
+id: avatar
+
+title: Avatar
+
+slug: /avatar
+---
+
+import Usage from './usage/Avatar/Avatar.md'
+
+Avatars are found all over ui design from lists to profile screens.
+
+They are commonly used to represent a user and can contain photos, icons, or even text.
+
+## Components
+
+- [Avatar.Accessory](#avataraccessory)
+  This is used for adding an accessory to the Avatar.
+  Receives either all [Icon](icon#props) or [Image](image#props) props.
+
+## Usage
+
+<Usage />
+
+---
+
+## Props
+
+### Avatar
+
+- [Component](#component)
+- [ImageComponent](#imagecomponent)
+- [activeOpacity](#activeopacity)
+- [avatarStyle](#avatarstyle)
+- [containerStyle](#containerstyle)
+- [icon](#icon)
+- [iconStyle](#iconstyle)
+- [imageProps](#imageprops)
+- [onLongPress](#onlongpress)
+- [onPress](#onpress)
+- [overlayContainerStyle](#overlaycontainerstyle)
+- [placeholderStyle](#placeholderstyle)
+- [renderPlaceholderContent](#renderplaceholdercontent)
+- [rounded](#rounded)
+- [size](#size)
+- [source](#source)
+- [title](#title)
+- [titleStyle](#titlestyle)
+
+### Avatar.Accessory
+
+- [style](#style)
+- [underlayColor](#underlaycolor)
+
+## Reference
+
+### Avatar
+
+#### Component
+
+Component for enclosing element (eg: TouchableHighlight, View, etc).
+
+| Type            | Default                  |
+| --------------- | ------------------------ |
+| React Component | TouchableOpacity or View |
+
+---
+
+#### ImageComponent
+
+Custom ImageComponent for Avatar.
+
+| Type | Default |
+| ---- | ------- |
+| any  | None    |
+
+---
+
+#### activeOpacity
+
+Opacity when pressed.
+
+| Type   | Default |
+| ------ | ------- |
+| number | None    |
+
+---
+
+#### avatarStyle
+
+Style for avatar image.
+
+| Type       | Default |
+| ---------- | ------- |
+| ImageStyle | None    |
+
+---
+
+#### containerStyle
+
+Styling for outer container.
+
+| Type               | Default |
+| ------------------ | ------- |
+| View style(Object) | None    |
+
+---
+
+#### icon
+
+Displays an icon as the main content of the Avatar. **Cannot be used alongside title**. When used with the `source` prop it will be used as the placeholder.
+
+| Type       | Default |
+| ---------- | ------- |
+| AvatarIcon | None    |
+
+---
+
+#### iconStyle
+
+Extra styling for icon component.
+
+| Type               | Default |
+| ------------------ | ------- |
+| Text Style(Object) | None    |
+
+---
+
+#### imageProps
+
+Optional properties to pass to the avatar e.g "resizeMode".
+
+| Type               | Default |
+| ------------------ | ------- |
+| ImageProps(Object) | None    |
+
+---
+
+#### onLongPress
+
+Callback function when long pressing component.
+
+| Type       | Default |
+| ---------- | ------- |
+| () => void | None    |
+
+---
+
+#### onPress
+
+Callback function when pressing component.
+
+| Type       | Default |
+| ---------- | ------- |
+| () => void | None    |
```

Comment:
It should be just `function`. The return type doesn't matter 
Pls fix this everywhere

Reply:
Bump